.travis.yml

```diff
@@ -1,7 +1,17 @@
-language: node_js
-node_js:
+matrix:
```

Comment:
I'm not sure if this is the right way to do this...I basically need two completely different languages, and reading the docs, it seems the matrix is the way to do this.

Reply:
And I didn't want to do it the way the ui-classic does it because they nvm install on demand, and I was trying to find a way to use the node travis image directly.